javascript/implementation/LogBackedStore.ts

```diff
@@ -223,12 +223,24 @@ export class LogBackedStore implements Store {
         if (!this.exclusive)
             await this.unlockFile();
         unlockingFunction();
-        return {
+        const chain = {
             medallion,
             chainStart,
             actorId: actorId || 0,
             claimTime,
         };
+        this.claimedChains.push(chain);
```

Comment:
The chain was previously not pushed to the list of claimedChains in the logbacked store. Was that intentional?

Reply:
No looks like it was an oversight.